cfgov/jinja2/v1/_includes/atoms/tagline.html

```diff
@@ -15,8 +15,7 @@
 <div class="a-tagline{{ ' a-tagline__large' if is_large else '' }}">
 	<span class="u-usa-flag"></span>
    <div class="a-tagline_text">
-      {{ _('An official website of the') }}
-      <span class="u-nowrap">{{ _('United States government') }}</span>
+      {{ _('This is a United States government website') }}
```

Comment:
What's the thinking behind abandoning preventing US gov from wrapping? I thought that was something we didn't want to happen. It's easier if we don't care how it wraps, but just checking on whether we want it that way.

Reply:
This is a good question! When a screenreader reads that bit, it inserts a pause between the div and the span, which kinda sounds funky.

Moreover, `This is a United States government website` at the `large` tagline size in the footer doesn't break at 375px which is on the lower end of common viewport widths. This is essentially a trade off between screenreader performance and how that `large` tagline looks in the footer on especially small phones.